src/components/node-list/styles/_row.scss

```diff
@@ -119,6 +119,9 @@
 .pipeline-row__toggle {
   position: absolute;
   right: $row-padding-x;
+}
+
+.pipeline-row__toggle--cursor {
   cursor: pointer;
```

Comment:
See comment above about using pointer, but otherwise I think this might better with a semantic class name. Would the existing class `pipeline-nodelist__row--kind-toggle` as used below work here? Otherwise maybe something like `.pipeline-row__toggle--not-tag`.

Reply:
yea this is a fair point - The main reason I separated this out as a new class and named it under `--cursor` instead of directly applying it under the `-toggle` class is mainly for readability, to allow a clear indication of this class simply applying a cursor interaction to the element. 

Imo the css class naming should be separated from the logic that governs the application of this class ( i.e the logic, `not-tag`, is applied on the JS side), which allows flexibility in case the logic of applying the cursor is changed in the future, while the semantics of this naming allows it to be easily understood as applying a cursor css element to whatever logic we applied on the JS side. 

again, happy to discuss and let me know your thoughts otherwise :D

Reply:
I think at the moment the approach in use is semantic BEM style names with modifiers, rather than utility classes like `--cursor`, but @richardwestenra can confirm. At least for consistencies sake, I think this should come under `pipeline-nodelist__row--kind-toggle` but check with him I'd say.

Reply:
sure I can move this under the `--toggle` class for simplicity

Reply:
@bru5 On having moved the `pointer:cursor` setting under `pipeline-nodelist__row--kind-toggle`, unfortunately the cursor effect does not apply, and on having played around with the other classes, the cursor classes cannot be applied either unless it's an additional add-on class in the execution that I did.

In aligning to your comment, I had changed the class name to `-not-tag` as per your suggestion.